modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -104,6 +106,8 @@
 
 			&-text {
 				width: 100%;
+				display: flex;
+			  	align-items: center;
```

Comment:
Please use css variables consistently. Why we don't have a css selector here, but we do use it for the icon?

Reply:
In this case, I didn't use variables because there is no specification that this will be changed. 
The text will always be centered.